src/Compiler/Facilities/prim-lexing.fs

```diff
@@ -325,6 +325,9 @@ and [<Sealed>] internal LexBuffer<'Char>
         with get () = bufferAcceptAction
         and set v = bufferAcceptAction <- v
 
+    member val private SourceTextBuilder = System.Text.StringBuilder()
```

Comment:
Put data to rest of the fields above of this type.
The style here is - declaration, fields, private let functions, then members.

Make it obvious by design that the StringBuilder is being cleared when appropriate (where does buffer cleanup happen now when lexes moves to a different file?)